salt/meta/backupninja.yml

```diff
@@ -0,0 +1,5 @@
+backup:
+  salt:
+    fs_includes:
+    - /srv/salt/reclass/nodes/_generated
```

Comment:
IMO you should backup whole /srv/salt/reclass

Reply:
I think this isn't necessary, because usually classes are from git repository.

Reply:
I know but it doesn't have to, so I would prefer to backup all.